xformers/factory/weight_init.py

```diff
@@ -92,35 +88,6 @@ def _small_init_(tensor: torch.Tensor, gain: float = 1.0) -> torch.Tensor:
     return _no_grad_uniform_(tensor, -a, a)
 
 
-def _variance_scaling(tensor, scale=1.0, mode="fan_in", distribution="normal"):
```

Comment:
I forgot to remove that from the init PR @dianaml0, it's not being used and degraded the code cov, sorry about that. Unrelated to this PR